### ISSUE TITLE
[BUG] crc bundle generate support for macOS

### ISSUE THREAD

**Issue by jhcook** [code-ready/crc#3301]
### General information

  * OS: macOS
  * Hypervisor: vfkit
  * Did you run `crc setup` before starting it: Yes
  * Running CRC on: Desktop

## CRC version
```bash
$ crc version
CRC version: 2.6.0+e7251680
OpenShift version: 4.10.22
Podman version: 4.1.0
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
$ sw_vers 
ProductName:    macOS
ProductVersion: 12.5
BuildVersion:   21G72
```

### Steps to reproduce

  1. `crc bundle generate`

### Expected

Bundle created

### Actual

```bash
$ crc bundle generate
INFO Removing user's pull secret from instance disk and from cluster secret... 
INFO Waiting for user's pull secret removed from instance disk... 
INFO Stopping kubelet and all containers...       
INFO Stopping the instance, this may take a few minutes... 
INFO Copying the disk image to crc_vfkit_4.10.22_amd64_1660305815 
Not implemented for darwin
```

### Logs

N/A

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
@jhcook As of now this feature is only for linux, any specific task you have in mind to have this feature for mac?

**Comment by jhcook** [comment]
@praveenkumar yes I would like to create bundles for the exact same reason you would create them on Linux -- to save time recreating specific versions of the bundle aka gold build.

**Comment by gbraad** [comment]
This would need for libvirt on macOS or a need cluster provider to communicate with the hypervisor on mac. This would need support for the fw_cfg parameter that ignition relies on... and therefore, nearly impossible with the native driver, except for using qemu. 

Alternatively you can base your custom image on the image provided, using `crc generate`.

**Comment by jhcook** [comment]
Ok @gbraad I will have a look into this. The biggest hurdle I'm seeing with OCP Local is regenerating certificates and loading the operators. It takes forever to do this and minimises the value this product provides. I know the documentation mentions the workaround for the certificates by getting the latest `crc`, but this should be unnecessary. Anyhow, my primary goal at this point is generating base builds to increase development velocity.

**Comment by gbraad** [comment]
Regenerating certificates happens automagically. @praveenkumar might spend some time later to see if there is a work around. If you have ideas...

**Comment by gbraad** [comment]
The bundle command was added to allow a regenerating, that has updated certificates or added images, such as for use in a dev environment or CI.

**Comment by jhcook** [comment]
@gbraad that's great! Is there documentation on this capability I've missed?

**Comment by gbraad** [comment]
The documentation is at the moment not in the best state. A backlog and s departure of the writer to a new team

**Comment by cfergeau** [comment]
> @jhcook As of now this feature is only for linux, any specific task you have in mind to have this feature for mac?

Looking at https://github.com/code-ready/crc/blob/main/pkg/crc/machine/generate_bundle_linux.go it seems the macOS implementation could use the same code with `cp -c` instead of `qemu-img`
